src/Form/EventFormType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\CallbackTransformer;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use App\Entity\EventType;
+use App\Entity\EventCategory;
+use App\Entity\Country;
+
+class EventFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('name', TextType::class)
+            ->add(
+                'type',
+                EntityType::class,
+                [
+                    'class' => EventType::class,
+                    'choice_label' => 'name'
```

Comment:
@daria-sieroshtan 
I suggest to add default value

Reply:
@daria-sieroshtan 
This is not a replacement.
You probably want some particular type to be shown by default, to save your user's clicks